Fix/hintboxes

Thiemann96 opened the pull request:
Put some styling on the hint box also updated all docu links for the sensors 

![grafik](https://github.com/user-attachments/assets/cfe10628-5b81-4d49-8f4c-5dd1bbc6a776)


Closes #372

BjoernLuig commented:
I found a few minor things
- There is still a console.log of the TooltipViewer
- the new box seems disaligned at the bottom, has another corner roundnes then the other objects and its background color seems out of place (personal opinion)
![grafik](https://github.com/user-attachments/assets/08fe2ac4-aab8-4e99-b13d-108a008a3a75)
![grafik](https://github.com/user-attachments/assets/2b1de8c8-5e6c-4460-b589-a0222e0f7820)
- switching to english language does not apply to all components
![grafik](https://github.com/user-attachments/assets/55866970-cb63-431b-a25d-5ce0de0bcb57)
- after reload the english title is missing and the button is still in german
![grafik](https://github.com/user-attachments/assets/33362520-4f39-4a01-8194-ecba77cd689a)

Thiemann96 commented:
I added english tarnslations, adjusted corner roundness and background color. 

However, had my difficulties with the layout (making it aligned with the blockly window) and could not seem to get it to work without restructuring the layout completely. Would like to postpone this to a later date, if you have any idea how to do it, feel free to suggest something, otherwise I would merge and pick it up at a later date

BjoernLuig commented:
Looks good na dcan be merged in my opinion
